less/middle-content.less

```diff
@@ -0,0 +1,320 @@
+.home {  
+    margin-left: 7%;
+    margin-right: 7%;
+
+    @media @phone {
+      margin-top: 10%;
+    }
+  
+    .middle-content {    
+      height: 100vh;
+      width: 100%;
+      display: flex;
+      justify-content: space-around;
+      flex-flow: row wrap;
+      padding-bottom: 80%;
```

Comment:
This padding is another issue. Do you really want 80% padding? Probably not. You might want a small margin (a few rem maybe) to separate this portion of the content from the rest of the content, though.

Also -- the contents of this div are stacked on each other. Why set it to be a row and wrap it? Just set it to column and they'll stack for you.